consul_exporter.go

```diff
@@ -86,6 +86,15 @@ func NewExporter(uri string) *Exporter {
 			[]string{"service", "node"},
 		),
 
+		nodeChecks: prometheus.NewGaugeVec(
+			prometheus.GaugeOpts{
+				Namespace: namespace,
+				Name:      "agent_check",
+				Help:      "Is this check passing on this node?",
```

Comment:
I completely agree. I ran into this exact issue when trying out my patch a few days ago and have been scratching my head on the right approach.

There are other states, but I think consistency w/ the rest of the exporter is probably more important than capturing all the states. The rest of the exporter reports back passing and not-passing. If the user needs to see more states, consul has a dashboard too.